snippets/bubbleSort.md

```diff
@@ -0,0 +1,37 @@
+###bubbleSort
```

Comment:
`bubbleSort` is more of an algorithmic implementation than an everyday-use snippet, as there is already a native function built into JS. This snippet can be merged, but will have to be moved to the archive.

Reply:
Alright